OOP/Restaurant/item.py

```diff
@@ -0,0 +1,164 @@
+"""Module Contains classes Item, OrderedITem, Stock, Order, Payment  and Delivery from a 
```

Comment:
The same `item.py` module has inside `Delivery`, `Payment`, `Order`,  `Stock`,  `OrderedItem` classes.